docker/local/fetch_and_run.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Source https://github.com/awslabs/aws-batch-helpers/blob/master/fetch-and-run/fetch_and_run.sh
+# Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script can help you download and run a script from S3 using aws-cli.
```

Comment:
What is the purpose of this file?

Reply:
Mentioned in the PR descirption

>We add fetch_and_run.sh as the command for the local image, to allow easy execution of arbitrary scripts through AWS Batch etc. See https://aws.amazon.com/blogs/compute/creating-a-simple-fetch-and-run-aws-batch-job/ for details

For someone not using AWS Batch this should not make a difference in how they use the local containers, but we can use this in Batch to pull scripts from S3 and run them. Currently any script needs to pre-exist on the image, which means that we need to pull a new image every time we modify e.g. a training script.